README.md

```diff
@@ -3,5 +3,35 @@ A python bot framework for slack
 
 [![Build Status](https://travis-ci.org/arcticfoxnv/slackminion.svg?branch=master)](https://travis-ci.org/arcticfoxnv/slackminion) [![Code Climate](https://codeclimate.com/github/arcticfoxnv/slackminion/badges/gpa.svg)](https://codeclimate.com/github/arcticfoxnv/slackminion) [![Test Coverage](https://codeclimate.com/github/arcticfoxnv/slackminion/badges/coverage.svg)](https://codeclimate.com/github/arcticfoxnv/slackminion/coverage)
 
-# Documentation
+## Running
```

Comment:
Feel free to modify any of this... these are just my thoughts on how the app should be installed/used/developed.